core/src/main/java/org/apache/iceberg/hadoop/Util.java

```diff
@@ -59,9 +59,17 @@ public static FileSystem getFs(Path path, Configuration conf) {
     }
   }
 
+  /**
+   * @deprecated since 1.7.0, will be removed in 2.0.0.
+   */
```

Comment:
```suggestion
  /**
   * @deprecated since 1.8.0, will be removed in 1.9.0; use {@link
   *     Util#blockLocations(ScanTaskGroup, Configuration)} instead.
   */
```